src/library/scala/collection/mutable/Map.scala

```diff
@@ -135,15 +135,15 @@ trait MapOps[K, V, +CC[X, Y] <: MapOps[X, Y, CC, _], +C <: MapOps[K, V, CC, C]]
    *  multiple times, or may evaluate `op` without inserting the result.
    *
    *  @param  key the key to test
-   *  @param  op  the computation yielding the value to associate with `key`, if
+   *  @param  defaultValue  the computation yielding the value to associate with `key`, if
    *              `key` is previously unbound.
    *  @return     the value associated with key (either previously or as a result
    *              of executing the method).
    */
-  def getOrElseUpdate(key: K, op: => V): V =
+  def getOrElseUpdate(key: K, @deprecatedName("op", since="2.13.13") defaultValue: => V): V =
```

Comment:
The change itself looks straightforward (almost trivial) and reasonable. I miss information (in the PR, not in the code) about how does it affect binary compatibility (both backward and forward?

Reply:
If MiMA doesn't complain, then parameter metadata via reflection is not a binary compat issue. Binary sig represents only types.